.drone.yml

```diff
@@ -5,12 +5,14 @@ pipeline:
     environment:
       - DOCKER_HOST=tcp://127.0.0.1:2375
       - DOCKER_IMAGE=lev-web
+      - DOCKER_TEST_IMAGE="${DOCKER_IMAGE}-testing"
     commands:
       - docker build -t "${DOCKER_IMAGE}" .
-      - docker build -t "${DOCKER_IMAGE}-testing" -f Dockerfile-testing .
+      - docker build -t "${DOCKER_TEST_IMAGE}" -f Dockerfile-testing .
       - docker run -d --net host "${DOCKER_IMAGE}"
       - sleep 5
-      - docker run --net host "${DOCKER_IMAGE}-testing"
+      - docker run --net host "${DOCKER_TEST_IMAGE}"
+      - docker cp "${DOCKER_TEST_IMAGE}":/app/test.log ./build-tests.log
```

Comment:
Why not just `docker run --net host "${DOCKER_TEST_IMAGE}" > ./build-tests.log 2>&1`?